metabase/metabase
# (3/n) update `UndoListing` to match new design for bulk collection actions

Epic: https://github.com/metabase/metabase/issues/28737


### Description

Third in a series of PR updating the bulk collection actions (move and archive) flow. ([Previous PR](https://github.com/metabase/metabase/pull/28797))

This PR centers the undo toast to match the position of the bulk actions toast.

### How to verify

1. Click on a collection in the sidebar.
2. Click on checkboxes to select items.
3. Either undo or archive.
4. Verify action took place and undo toast now appears in center.
5. Verify clicking undo reverts the action.

### Demo

https://user-images.githubusercontent.com/37751258/222258012-2c9ac52d-eb3b-4e79-913d-47d2da9ca4cb.mov

### Checklist

- [✅] Tests have been added/updated to cover changes in this PR
